cloudformation/hello-world/env0.yml

```diff
@@ -0,0 +1,17 @@
+version: 1
+
+deploy:
+  onSuccess:
+    - echo Replacing !!!USER!!! with $USER in index.html
+    - sed 's/!!!USER!!!/'"$USER"'/g' index.template.html > index.html
+    - awsv2 configure set default.region ${AWS_REGION}
```

Comment:
probably don't need this if we set `AWS_DEFAULT_REGION`

Reply:
e8e8597aad018027993295b3f0b6b1416137f8e0